assets/js/components/notifications/SetupSuccessBannerNotification.js

```diff
@@ -169,10 +173,6 @@ function SetupSuccessBannerNotification() {
 				}
 			}
 
-			if ( serviceSetupV2Enabled ) {
-				winData.description = '';
-			}
```

Comment:
The changes in this file seem to have gotten here accidentally? They seem unrelated.